test/test_onnx.py

```diff
@@ -82,16 +82,29 @@ def to_numpy(tensor):
                     raise
 
     def test_nms(self):
-        boxes = torch.rand(5, 4)
-        boxes[:, 2:] += torch.rand(5, 2)
-        scores = torch.randn(5)
+        num_boxes = 100
+        boxes = torch.rand(num_boxes, 4)
+        boxes[:, 2:] += 1 + torch.rand(num_boxes, 2)
```

Comment:
for a better fix, can you do as @BowenBao proposed and do
```python
boxes[:, 2:] += boxes[:, :2]
```
?
This way we ensure that the boxes are always well-formed (with potentially some boxes being 0, but we can add an eps to avoid that). The current fix works because `rand` is between 0-1, but if we were to have a different distribution we would have similar problems (and I was bitten by that in the past)